common.py

```diff
@@ -4,7 +4,7 @@
 from typing import List
 
 FT_BASE_IMG = "quay.io/wizhao/ft-base-image:0.9"
-TFT_TOOLS_IMG = "quay.io/wizhao/tft-tools:09-27-2023"
+TFT_TOOLS_IMG = "quay.io/wizhao/tft-tools:0.3"
```

Comment:
If the github actions automation is working as expected, we should be using the latest image rather than a hardcoded version